pkgs/racket-doc/scribblings/reference/sequences.scrbl

```diff
@@ -595,7 +595,7 @@ each element in the sequence.
 @defproc[(in-directory [dir (or/c #f path-string?) #f]
                        [use-dir? ((and/c path? complete-path?) . -> . any/c)
                                  (lambda (dir-path) #t)])
-         sequence?]{
+         (sequence/c path?)]{
```

Comment:
Should these all be `path-for-some-system?` instead?

Reply:
I think `path?` is right everywhere, since the paths come from the current filesystem, so they're definitely for the current platform.